src/components/ToDoForm.js

```diff
@@ -11,18 +11,21 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function ToDoForm({ inputText, handleChange, handleSubmit }) {
+function ToDoForm({ inputText, handleChange, handleSubmit, editingItem }) {
```

Comment:
```suggestion
function ToDoForm({ inputText, handleChange, handleSubmit, itemToEdit }) {
```
`itemToEdit` makes more sense when reading code.